### ISSUE TITLE
First invocation on mac in fullscreen

### ISSUE THREAD

**Issue by bdillahu** [tumashu/ivy-posframe#2]
I have emacs saved as fullscreen in my desktop file.

When you start emacs it opens fullscreen on the Mac, hiding the menu bar and all that. Then I hit M-x and it shrinks the Mac emacs window down to "default" size, as if it wasn't fullscreen. It leaves the rest of the screen black, which is weird.

Things still work fine. If you hit the green "maximize" button again, it maximizes and after that seems happy.

Here's my settings:
    (setq ivy-display-function #'ivy-posframe-display)
    (setq ivy-posframe-style 'window-buttom-left)

**Comment by tumashu** [comment]
Maybe, child-frame has more bug on Mac,  try the below code:

```
(posframe-show " *test*"
               :string "this is a test")
```

**Comment by bdillahu** [comment]
I'm sorry, where should I be trying this?

**Comment by tumashu** [comment]
launch emacs  -> funscreen ->  eval the above code in a buffer

**Comment by bdillahu** [comment]
(posframe-show " *test*"
               :string "this is a test")
nil
this is a test

**Comment by tumashu** [comment]
```
When you start emacs it opens fullscreen on the Mac, hiding the menu bar and all that. Then I hit M-x and it shrinks the Mac emacs window down to "default" size, as if it wasn't fullscreen. It leaves the rest of the screen black, which is weird.
```
Test the problem, I will test whether it is a posframe's problem

**Comment by bdillahu** [comment]
Well, I'm not sure if I sent you down the wrong path or not... I managed to make two changes at once so I don't know which fixed it, but it has quit happening.

I deleted my .emacs.desktop file and redid that...

And I happened to restart and update packages and it pulled a newer version of your code... so not sure what your last change had, but maybe I had missed the latest.

I'll let you know if I see it again and can figure anything out to narrow it down.

Thanks!
